src/client/utils/tracking/gtm.ts

```diff
@@ -120,13 +128,14 @@ export const trackOfferGTM = (
         has_accident: hasAccidentQuote(offerData),
         has_travel: hasTravelQuote(offerData),
         initial_offer: initialOffer,
```

Comment:
Instead of reassigning `initialOffer` on line 114 - couldn't you just use different variables depending on whether we have `initialOffer` or not? Like
```suggestion
        initial_offer: initialOffer ? initialOffer : contractCategory,
```
```suggestion
        initial_offer: initialOffer || contractCategory,
```
Or am I missing something? Probably (lol) 🙈

Reply:
I'm open for suggestions!  :) Yes, we can do it like that for sure if it feels clearer  🙌